Makefile

```diff
@@ -2,8 +2,8 @@
 
 # Command to run pytest for correctness tests
 test:
-	pytest --disable-warnings test/ --ignore=test/convergence
-	
+	python -m pytest --disable-warnings test/ --ignore=test/convergence
```

Comment:
I found that this command wasnt working correctly out of the box - it was failing to resolve the `test.utils` module. Using `python -m` ensures that the root dir of the project is added to the python path so that `test.utils` is resolved.

Reply:
awesome